.github/workflows/ci.yaml

```diff
@@ -20,7 +22,16 @@ jobs:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
       - name: install buf
         uses: bufbuild/buf-setup-action@v1.7.0
-      - name: install rust-toolchain
+      - name: install protoc
```

Comment:
how about using this action instead?
https://github.com/marketplace/actions/setup-protoc